index.js

```diff
@@ -1,4 +1,8 @@
 /* jshint node: true */
+const path = require('path');
+const Funnel      = require('broccoli-funnel');
```

Comment:
Should we used `var` rather than `const` here so we don't lose support for older node versions? In general im OK with using the latest but maintaining support here seems easy enough :)